assets/src/containers/IndexPage.js

```diff
@@ -35,20 +25,16 @@ function IndexPage (props) {
   }
 
   return (
-    <Grid container className={classes.root}>
-      <Grid item xs={12} className={classes.container}>
-        <Grid
-          container
-          className={classes.wrapper}
-          spacing={8}
-        >
-          {routes(courseId, views).map((props, key) =>
-            <Link style={{ textDecoration: 'none' }} to={props.path} key={key}>
+    <Grid container spacing={2}>
+      {
+        routes(courseId, views).map((props, key) => (
+          <Grid item xs={12} sm={6} lg={4} key={key}>
```

Comment:
I'd be interested to hear what others think, but I think it'd be nice to limit the maximum width (maybe to just longer than the length of explanatory text?) for the cards. When you're full screen, they just seem to me wider than necessary for their content.

Reply:
We should have a consistent width for all views, or it'll look strange. We should adjust the width by playing around with the margin or having 4 views fit on one row. I personally think having 4 looks a bit crowded.

Reply:
You're right, I wasn't suggesting using different widths for the different cards; they should be consistent. I also agree that having four cards across in the widest windows might look awkward. I was just trying to say that I think it looks awkward to have the cards as wide as they get, given the shortness of explanatory text and the size of the screenshots.

Reply:
Agreed, they do look rather large. Maybe using different sized margins might help, or adding more explanatory text?

Reply:
I don't think we can adjust the explanatory text. Is setting a fixed maximum width for all cards not an option?

Reply:
Not exactly. The width gets dynamically calculated based on the number of cards we want to fit on a row. If we set a maximum width for a card, then there potentially could be a bunch of empty space between the cards on the same row, depending on the width of the browser.

Reply:
I went through the discussion comment here, The Card view is widened so It really doesn't matter for me w.r.t this PR it's ok to merge. But I will ask for input from our style guide analyst, @johnpj can you review this work I put this branch on [dev](https://dev-myla.tl.it.umich.edu/) instance